### ISSUE TITLE
--cvars does not work for JVM_SYS_PROP

### ISSUE THREAD

**Issue by ollis** [laetho/metagraf#7]
Given a metagraf with 
````
    "environment": {
      "local": [
        {
          "name": "JAVA_OPTIONS",
          "required": true,
          "type": "JVM_SYS_PROP",
          "description": "Configured Java system properties"
        },
````
And
```
    "config": [
      {
        "name": "JVM_SYS_PROP",
        "type": "JVM_SYS_PROP",
        "description": "JVM system properties",
        "options": [
          {
            "name": "my.test.prop",
            "required": true,
            "description": "test property",
            "type": "string",
            "default": "test_value"
          }
        ]
      }
    ]
```
I expect `mg create deploymentconfig  metagraf.json --cvars "my.test.prop=nissemann" --namespace cop --dryrun` to populate the JAVA_OPTIONS environment like this:
```
              {
                "name": "JAVA_OPTIONS",
                "value": "-Dmy.test.prop=nissemann"
              },
```
But actual result is this:
```
              {
                "name": "JAVA_OPTIONS"
              },
```

**Comment by laetho** [comment]
This is definatly a bug. I suspect it hides somewhere in the code that merges environment variables, --cvars and --cvfile inputs to addressable values in the metaGraf specification or in the pkg/modules/java.go file.

**Comment by laetho** [comment]
Working on this in PR #8. Turned into some refactoring work to make it all a bit more understandable.

**Comment by laetho** [comment]
This bug should now be resolved in PR #8. I'll get this merged and part of the mg v0.0.32 release.
